index.html

```diff
@@ -63,8 +63,117 @@ <h3 class="skill-title">Skills</h3>
         <div>
             <div class="eli18"><img src="images/icones/Ellipse18.svg" alt=" img ackround "></div>
             <div class="eli19"><img src="images/icones/Ellipse19.svg" alt="img backgound "></div>
+  
         </div>
     </section>
+    <div class="work">
+        <h3>WORK</h3>
+      </div>
+    <div class="recent-work">
+      <h2 class="recent">MY RECENT <br> WORK</h2>
+    </div>
+  <hr>
```

Comment:
As asked by the previous reviewer, correct the layout of the `<hr>`
You can use `margin-bottom: 50px` to increase the marge between the line and the image

<img width="810" alt="image" src="https://user-images.githubusercontent.com/7243132/171291283-86b1a3a8-8dfe-4bee-be1c-e781f8c22653.png">

Reply:
thank you!!